src/components/NotificationsAlert.tsx

```diff
@@ -12,20 +12,21 @@ import {
 } from '@/utils/notifications';
 import { isNotificationSupported, isPermisionGranted, isPushManagerSupported } from '@/utils/notificationsHelpers';
 import { getNotificationLocalStorage, setNotificationsSessionStorage } from '@/utils/notificationsLocalStorage';
-import type { TosData } from '@/utils/types';
+import type { TocData } from '@/utils/types';
```

Comment:
What does `Toc` stand for? I know `Tos => Terms of Service`. Just curious about the renaming.

Reply:
I was thinking about `Terms of Conditions => Toc`. But let's keep `Tos` then.